app/serializers/page_serializer/statutory_instruments_show_page_serializer.rb

```diff
@@ -19,47 +19,30 @@ def meta
-      end
-    end
-
-    def section_primary_components
-      [ComponentSerializer::Heading1ComponentSerializer.new({ subheading_content: 'statutory-instruments.show.subheading', heading_content: title, context_content: "#{@statutory_instrument.prefix} #{@statutory_instrument.year}/#{@statutory_instrument.number}" }).to_h]
-    end
-
-    def section_literals
-      [].tap do |components|
-        components << ComponentSerializer::HeadingComponentSerializer.new(content: 'statutory-instruments.show.literals', size: 2).to_h
-        components << ComponentSerializer::ListDescriptionComponentSerializer.new(items: literals).to_h
-      end
+    def heading1_component
+      ComponentSerializer::Heading1ComponentSerializer.new({ subheading_content: 'statutory-instruments.show.subheading', heading_content: title, context_content: "#{@statutory_instrument.try(:statutoryInstrumentPaperPrefix)} #{@statutory_instrument.try(:statutoryInstrumentPaperYear)}/#{@statutory_instrument.try(:statutoryInstrumentPaperNumber)}" }).to_h
     end
 
-    def literals
+    def meta_info
       [].tap do |items|
+        items << create_description_list_item('laid-thing.web-link', ["<a href='#{@laid_thing.try(:workPackagedThingHasWorkPackagedThingWebLink)}'>#{@laid_thing.try(:workPackagedThingHasWorkPackagedThingWebLink)}</a>"])
         items << create_description_list_item('statutory-instruments.show.made-date', [l(@statutory_instrument.made_date)])
-        items << create_description_list_item('statutory-instruments.show.coming-into-force-note', [@statutory_instrument.coming_into_force_note])
+        items << create_description_list_item('laid-thing.laid-date', [l(@laid_thing&.laying&.date)])
         items << create_description_list_item('statutory-instruments.show.coming-into-force-date', [l(@statutory_instrument.coming_into_force_date)])
+        items << create_description_list_item('statutory-instruments.show.coming-into-force-note', [@statutory_instrument.try(:statutoryInstrumentPaperComingIntoForceNote)])
+        items << create_description_list_item('statutory-instruments.show.following-title', connected_statutory_instruments)
+        items << create_description_list_item('laid-thing.laying-person', [@laying_person&.display_name])
+        items << create_description_list_item('laid-thing.laying-body', [@laying_body.try(:groupName)])
       end.compact
     end
 
     def connected_statutory_instruments
       @preceding_proposed_negative_statutory_instruments.map do |stat_instrument|
-        ComponentSerializer::LinkComponentSerializer.new(
-          link:    proposed_negative_statutory_instrument_path(stat_instrument.graph_id),
-          content: stat_instrument.name
-        ).to_h
+        "<a href='#{proposed_negative_statutory_instrument_path(stat_instrument.graph_id)}'>#{stat_instrument.try(:proposedNegativeStatutoryInstrumentPaperName)}</a>"
```

Comment:
We should talk about moving this to a link_to or equivalent helper, I'm not sure I like the html as a string here. Alternatively, maybe into the translations? Not needed for the initial release though 👍🏻